### PR TITLE
changed install directory so other pkgs can ament_target_depend on th…

### PR DESCRIPTION
**Pull request by buckleytoby**
…is library

### CODE REVIEW

**Comment by berndpfrommer** [comment]
The checks failed because the CI workflow file was pointing to the wrong yaml file. I pushed the necessary changes to the master but now your branch is behind by that commit. Could you please pull master and rebase to it, then push to your PR branch again so I can rerun the tests? Thanks!

**Comment by berndpfrommer** [comment]
@buckleytoby can you merge master and push to your PR branch again? It should trigger a rebuild so we know the tests pass. Thanks!

File: src/ffmpeg_encoder.cpp [hunk 1]
```diff
@@ -91,6 +91,7 @@ void FFMPEGEncoder::setParameters(rclcpp::Node * node)
   profile_ = get_safe_param<std::string>(node, ns + "profile", "");
   preset_ = get_safe_param<std::string>(node, ns + "preset", "");
   tune_ = get_safe_param<std::string>(node, ns + "tune", "");
+  delay_ = get_safe_param<std::string>(node, ns + "delay", "");
```
Comment (berndpfrommer) [inline src/ffmpeg_encoder.cpp:94]: Please update README.md to reflect this new parameter.

**Review by berndpfrommer — COMMENTED** [review]
LGTM but please add documentation to README.md.
Thanks!

**Review by berndpfrommer — APPROVED** [review]
Needs Documentation of  new parameter "delay" in README.md. Rest looks good.

**Comment by berndpfrommer** [comment]
Since I got no response from @buckleytoby I merged the change without any documentation :(

**Comment by buckleytoby** [comment]
hey, sorry, I've been sick all week so haven't had the chance to revisit this. Will return to it on Monday.

**Pull request merged**